### ISSUE TITLE
DUP: Duplicate of #124 - Input Editor: History Autosuggestions

### ISSUE THREAD

**Issue by classner** [warpdotdev/warp#339]
### Describe the solution you'd like?

```shell
I love zsh autosuggestions (see here: https://github.com/zsh-users/zsh-autosuggestions): they are incredibly helpful for two reasons: 1) if the last completion is the correct one (which is often the case), a single keystroke completes the suggestion. 2) even if it is not, it is helpful to see what it looked like to type an adapted version.

In warp, this feature could look slightly different since the entire UI is more powerful: it could do an auto-history search while typing while keeping the focus in the text field, displaying the history above (like the current CTRL-R functionality). Or other ways..?
```

**Comment by elviskahoro** [comment]
We added autosuggestions. I have your specific feedback noted though for when we work on a V2. Closed: v0.2021.11.09.19.46.stable_00

**Comment by classner** [comment]
@elviskahoro That's great, thanks a lot!
